okteto/okteto
# refactor: remove okteto context global var use in okteto build command

this PR is intended to inject the okteto context values into the build command

### Changes
- create `pkg/build/context.go` to hold the struct to inject in the build command
- read context before start build command in order to inject okteto context
### How to test
- check `build` command works as expected. No panics detected or weird behaviors.
